Auto update failing for ipfs

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [ipfs](https://github.com/termux/termux-packages/blob/f7ef965ee88fdfdad18685dee01234cbd3a66e48/packages/ipfs) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating ipfs [Current version: 0.29.0]
INFO: package being updated to 0.30.0.
Downloading https://github.com/ipfs/kubo/releases/download/v0.30.0/kubo-source.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0 15.7M    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 15.7M  100 15.7M    0     0  15.3M      0  0:00:01  0:00:01 --:--:-- 40.7M

You are about to commit these changes:

--------------------
diff --git a/packages/ipfs/build.sh b/packages/ipfs/build.sh
index ff915c732..d1bd5a6f0 100644
--- a/packages/ipfs/build.sh
+++ b/packages/ipfs/build.sh
@@ -3,9 +3,9 @@ TERMUX_PKG_DESCRIPTION="A peer-to-peer hypermedia distribution protocol"
 TERMUX_PKG_LICENSE="MIT, Apache-2.0"
 TERMUX_PKG_LICENSE_FILE="LICENSE, LICENSE-APACHE, LICENSE-MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.29.0"
+TERMUX_PKG_VERSION="0.30.0"
 TERMUX_PKG_SRCURL=https://github.com/ipfs/kubo/releases/download/v${TERMUX_PKG_VERSION}/kubo-source.tar.gz
-TERMUX_PKG_SHA256=b9d095c80dcd377442995b5d22371c7f2fd1ca6bebb06268c6517c0e26a93f88
+TERMUX_PKG_SHA256=f863b6dc0f431c81ee2b318d90f2c136fbcfcc52f715f418458bbd4df3e28b4f
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_SUGGESTS="termux-services"
 TERMUX_PKG_SERVICE_SCRIPT=("ipfs" "[ ! -d \"${TERMUX_ANDROID_HOME}/.ipfs\" ] && ipfs init --empty-repo 2>&1 && ipfs config --json Swarm.EnableRelayHop false 2>&1 && ipfs config --json Swarm.EnableAutoRelay true 2>&1; exec ipfs daemon --enable-namesys-pubsub 2>&1")
--------------------

bump(main/ipfs): 0.30.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package ipfs exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building ipfs for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   163k      0 --:--:-- --:--:-- --:--:--  164k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  17615      0 --:--:-- --:--:-- --:--:-- 17723
gpg: Signature made Wed 11 Sep 2024 02:50:10 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0   212k      0 --:--:-- --:--:-- --:--:--  213k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  13731      0 --:--:-- --:--:-- --:--:-- 13883
gpg: Signature made Tue 10 Sep 2024 06:29:13 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0   254k      0 --:--:-- --:--:-- --:--:--  256k
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  15267      0 --:--:-- --:--:-- --:--:-- 15425
gpg: Signature made Wed 11 Sep 2024 12:21:09 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://github.com/ipfs/kubo/releases/download/v0.30.0/kubo-source.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 15.7M  100 15.7M    0     0  31.6M      0 --:--:-- --:--:-- --:--:-- 31.6M
Applying patch: go-ipfs-0.4.15_bin_dist_get.patch
go version go1.23.1 linux/amd64
bin/check_go_version 1.18
plugin/loader/preload.sh > plugin/loader/preload.go
go fmt plugin/loader/preload.go >/dev/null
go build "-trimpath" -ldflags="-X "github.com/ipfs/kubo".CurrentCommit=846c5ccf6" -o "cmd/ipfs/ipfs" "github.com/ipfs/kubo/cmd/ipfs"
# github.com/ipfs/kubo/cmd/ipfs
link: github.com/wlynxg/anet: invalid reference to net.zoneCache
make: *** [cmd/ipfs/Rules.mk:22: cmd/ipfs/ipfs] Error 1
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-09-11 18:19:20 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/10817146414">10817146414</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/ipfs/build.sh
+++ b/packages/ipfs/build.sh
@@ -3,12 +3,14 @@ TERMUX_PKG_DESCRIPTION="A peer-to-peer hypermedia distribution protocol"
 TERMUX_PKG_LICENSE="MIT, Apache-2.0"
 TERMUX_PKG_LICENSE_FILE="LICENSE, LICENSE-APACHE, LICENSE-MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.29.0"
+TERMUX_PKG_VERSION="0.30.0"
 TERMUX_PKG_SRCURL=https://github.com/ipfs/kubo/releases/download/v${TERMUX_PKG_VERSION}/kubo-source.tar.gz
-TERMUX_PKG_SHA256=b9d095c80dcd377442995b5d22371c7f2fd1ca6bebb06268c6517c0e26a93f88
+TERMUX_PKG_SHA256=f863b6dc0f431c81ee2b318d90f2c136fbcfcc52f715f418458bbd4df3e28b4f
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_SUGGESTS="termux-services"
 TERMUX_PKG_SERVICE_SCRIPT=("ipfs" "[ ! -d \"${TERMUX_ANDROID_HOME}/.ipfs\" ] && ipfs init --empty-repo 2>&1 && ipfs config --json Swarm.EnableRelayHop false 2>&1 && ipfs config --json Swarm.EnableAutoRelay true 2>&1; exec ipfs daemon --enable-namesys-pubsub 2>&1")
+# See https://github.com/ipfs/kubo/issues/10501 about issues with ipfs and go 1.23:
+TERMUX_PKG_GO_USE_OLDER=true
 
 termux_step_make() {
 	termux_setup_golang

